src/celeritas/optical/OpticalGenData.hh

```diff
@@ -42,7 +42,7 @@ struct OpticalGenParamsData
 
     bool cerenkov{false};  //!< Whether Cerenkov is enabled
     bool scintillation{false};  //!< Whether scintillation is enabled
-    real_type capacity{0};  //!< Distribution data buffer capacity
+    size_type capacity{0};  //!< Distribution data buffer capacity
```

Comment:
@amandalund I assume this was the intent, and that I'm not missing something?

Reply:
Yes, thanks, that should be `size_type`.